Subtle mistake in my openat(2) code

jawline opened the pull request:
I forgot we can just raise rather than core dumping, adjusted.

jawline committed:
Stubs

jawline committed:
Merge branch 'master' of github.com:jawline/io_uring

mt-caret commented on src/io_uring_stubs.c:
```diff
@@ -78,7 +78,9 @@ CAMLprim value io_uring_prep_open_stub(value v_io_uring, value v_sqe_flags, valu
-  assert(v_bstr_len >= sizeof(struct open_how));
+  if (v_bstr_len < sizeof(struct open_how)) {
+    uerror("bstr memory", Nothing);
+  }
```
I failed to notice this in your original PR, but I think you need `Long_val(v_bstr_len)` instead of just `v_bstr_len`.

jawline replied:
Sod I'm sure I wrote the originally. I must have refactored it out.

jawline committed:
Call Long_val on bstrlen

Previously this worked because Ocamls underlying representation of integer values is immediate (value ~= int63) but we shouldn't rely on this.
